api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -155,6 +155,10 @@ type AgentClusterInstallSpec struct {
 	// DiskEncryption is the configuration to enable/disable disk encryption for cluster nodes.
 	// +optional
 	DiskEncryption *DiskEncryption `json:"diskEncryption,omitempty"`
+
+	// Set to true to allow control plane nodes to be schedulable
+	// +optional
+	MastersSchedulable bool `json:"mastersSchedulable,omitempty"`
```

Comment:
Maybe MastersSchedulable -> ControlPlaneSchedulable

We should avoid 'master' terminology

Reply:
This is an official attribute of openshift and we should preserve that naming. see the link in the description. It describe this attribute and its use